openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -446,6 +449,10 @@ The request parameter `authorization_details` defined in Section 2 of [@!RFC9396
 * `credential_configuration_id`: REQUIRED when `format` parameter is not present. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format-profiles). It MUST NOT be present if `format` parameter is present.
 * `format`: REQUIRED when `credential_configuration_id` parameter is not present. String identifying the format of the Credential the Wallet needs. This Credential Format Identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format-profiles). It MUST NOT be present if `credential_configuration_id` parameter is present.
 
+Additional `authorization_details` data fields MAY be defined and used,
+as described in [@!RFC9396].
+The Credential Issuer MUST ignore any unrecognized parameters.
```

Comment:
I wonder if this extensibility is actually compliant with RAR. Section 5 of RFC 9396 states that:

> The AS MUST abort processing and respond with an error `invalid_authorization_details` to the client if [an object in the authorization_details structure] [...] is an object of known type but containing unknown fields,

Does the proposed extensibility mean that, when it comes to the `openid_credential` type, there are no "unknown fields" because by definition, any unrecognized field gets ignored and consequently all fields are expected and thus "known"?

Reply:
Ouch, well spotted, I'd missed that. That's definitely awkward...

Reply:
Thanks for pointing out this conflict, @ju-cu.  I added language to the PR explicitly addressing this conflict.

Reply:
why do we want to overwrite RAR language here though? I think we should rather stick to RAR and remove 
the following sentence:

> Additional `authorization_details` data fields MAY be defined and used
> when the `type` value is `openid_credential`.

Reply:
We're deviating from RAR for this type because that enables non-breaking extensibility when using the type `openid_credential`.  Without this language, if RAR if followed strictly, if a new `authorization_details` parameter is used, it would cause an error in existing deployments.  Meaning that this type would not be extensible.

Reply:
@jogu's proposal is to define a claim for this authorization_details type as an alternative extensibility point

Reply:
@tlodderstedt WDYT?

Reply:
I don't think anything is being overwritten. If the openid_credential type is defined such that it can have additional data fields, then by definition it can have additional data fields. I think this can probably be noted with language that's less likely to be objectionable.